.circleci/config.yml

```diff
@@ -55,8 +55,8 @@ jobs:
         conda activate base &&
         cd ${ILASTIK_ROOT}/ilastik-meta &&
         python ilastik/scripts/devenv.py create -n ${TEST_ENV_NAME}
-        -p pytest-qt ilastik-dependencies-no-solvers black
-        -c ilastik-forge conda-forge defaults
+        -p pytest-qt ilastik-dependencies-no-solvers black elf
```

Comment:
@k-dominik I cannot figure out why CircleCI cannot find elf (hence the tests fail) once a test tried to import it. Any clue? I thought adding it as a dependency here would make it readily available to the environment.